config/more_samples-glyf_colr_1.py

```diff
@@ -1115,6 +1141,33 @@ def main():
         _gradient_p2_skewed(next(access_chars)),
         _colrv0_colored_circles(palette_test_colors, next(access_chars)),
         _colrv1_colored_circles(palette_test_colors, next(access_chars)),
+        # Sweep with repeat mode pad
+        _sample_sweep(0, 90, "pad", "narrow", next(access_chars)),
+        _sample_sweep(45, 90, "pad", "narrow", next(access_chars)),
+        _sample_sweep(247.5, 292.5, "pad", "narrow", next(access_chars)),
+        _sample_sweep(90, 270, "pad", "narrow", next(access_chars)),
+        _sample_sweep(-270, 270, "pad", "narrow", next(access_chars)),
+        _sample_sweep(-45, 45, "pad", "narrow", next(access_chars)),
+        _sample_sweep(315, 45, "pad", "narrow", next(access_chars)),
```

Comment:
should you also test a case where start angle > end angle (and does not wrap around 360), e.g. start=90, end=45? I'd espect the sweep to proceed clock-wise

Reply:
> I'd espect the sweep to proceed clock-wise

no, actually sweep always goes counter-clockwise by definition

https://docs.microsoft.com/en-us/typography/opentype/spec/colr#sweep-gradients